### ISSUE TITLE
fix aria-hidden text

### ISSUE THREAD

**Issue by jnurthen** [w3c/aria#1618]
aria-hidden states

> User agents determine an element's hidden status based on whether it is rendered, and the rendering is usually controlled by CSS. For example, an element whose display property is set to none is not rendered. An element is considered hidden if it, or any of its ancestors are not rendered or have their aria-hidden attribute value set to true.

This is incorrect - visibility:hidden *is* rendered per HTML's definition ( https://html.spec.whatwg.org/multipage/rendering.html#being-rendered ) but is hidden, - and children of visibility:hidden can be set to visible and be rendered AND visible - so clearly not hidden.

**Comment by chlane** [comment]
I would like to work on this one. It seems we need to also explain that rendering and visibility are separate concepts.

**Comment by chlane** [comment]
@jnurthen since rendering doesn't have anything to do with hidden status can we just say " An element is considered hidden if it, or any of its ancestors have their aria-hidden attribute value set to true"?
